src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
```

Comment:
It seems like your span elements are used to represent stars. Consider adding some text inside these elements or use aria-label attribute to describe the content for screen readers. It's important to make your website accessible for all users.